### PR TITLE
Fixing bug in network sync

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/node-agent#674

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Tested network details are coming properly

**Comment by GowthamShanmugam** [comment]
@r0h4n @nthomas-redhat please review

File: tendrl/node_agent/node_sync/network_sync.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def get_node_network():
     network_interfaces = get_node_interface()
     cmd = cmd_utils.Command('hwinfo --network')
     out, err, rc = cmd.run()
-    if not err:
+    if (not err) or "vdsmdummy: command not found" in err:
```
Comment (r0h4n) [inline tendrl/node_agent/node_sync/network_sync.py]: you dont need the round brackets

Comment (GowthamShanmugam) [reply tendrl/node_agent/node_sync/network_sync.py]: ok

**Comment by GowthamShanmugam** [comment]
@r0h4n changes done

**Pull request merged**